addons/hr_recruitment/models/hr_applicant.py

```diff
@@ -493,6 +503,57 @@ def name_get(self):
                 for applicant in self
             ]
         return super().name_get()
+    @staticmethod
+    def _parse_ict_job_application(msg):
+        tree = html.fromstring(msg.get('body'))
+        text_elements = tree.xpath("//text()")
+        mail_index = 0
+
+        for text in text_elements:
+            if "@" in text:
+                break
+            mail_index += 1
+        try:
+            email = text_elements[mail_index]
```

Comment:
Were you able to test this on multiple emails, so do we know that the order of these items will always be exactly like this?